**Transcriptomics in Cancer Biomarker Discovery**

Genes that carry information for functional products like proteins or noncoding RNAs are transcribed to mRNA, and these transcripts are used to make the corresponding products. To study the transcriptome, the bulk of all the genes being expressed at any given time in a cell, tissue, or organism, transcriptomics was born (Horgan and Kenny, 2011). To study how the genes that affect diseases like cancer are expressed, transcriptomics is important (Lowe et al., 2017). Through it, we can understand the differences in gene expression between cells or tissues, for example, normal versus diseased (Fan et al., 2020).

Transcriptomics has been useful in bringing to light cancer biomarkers (Srivastava, 2022). Transcriptomics makes it possible to compare the expression of genes in normal and cancer cells, and the genes differentially expressed in these cells could be biomarkers for the disease (Giulietti _et al_., 2018). Biomarkers are especially useful in the diagnosis and detection of cancer tumors, as well as in the monitoring of cancer treatment progress (Kosaka _et al_., 2010).

Although there are different techniques for transcriptome analysis, one that stands out is RNA sequencing, also called RNA-seq (Van Rensburg _et al_., 2015). RNA-seq, as the name suggests, is a sequencing-based technique that allows for accurate measurement of gene expression by quantifying the amount of cDNA made from transcripts at a remarkable level of precision (Van Rensburg _et al_., 2015; Corchete _et al_., 2020). RNA-seq has helped identify tumorigenic mutations and cancer biomarkers (Puram _et _al., 2017; Nakagawa _et al_., 2019).

Transcriptomics has led to the discovery of various biomarkers for different cancer types. A study by Zhang _et al._ aimed to identify breast cancer biomarkers from saliva using transcriptomic techniques. A total of eight mRNA biomarkers present in the saliva of breast cancer patients were identified and validated in the study (Zhang _et al_., 2010). In a study on lung adenocarcinoma by Yang _et al_., four genes of the human leukocyte antigen family were identified with RNA-seq as biomarkers that could be targeted for treating the disease (Yang _et al_., 2014).

The major challenges in transcriptomic biomarker discovery include issues with transcriptomic data analysis during the quality control and normalization steps, clinical validation of identified biomarkers, and securing funds for transcriptomic biomarker research (Siemelink and Zeller, 2014). With the evolution of transcriptomic techniques into better and less expensive versions of their current selves, cancer biomarker discovery and validation will improve, and the challenges facing transcriptomic biomarker discovery can be phased out. As more biomarkers are found and validated, people can benefit from the fruits of transcriptomic biomarker research better and more than ever.

**References**

Corchete, L.A., Rojas, E.A., Alonso-López, D., De Las Rivas, J., Gutiérrez, N.C. and Burguillo, F.J., 2020. Systematic comparison and assessment of RNA-seq procedures for gene expression quantitative analysis. _Scientific reports_, _10_(1), p.19737.****

Fan, J., Slowikowski, K. and Zhang, F., 2020. Single-cell transcriptomics in cancer: computational challenges and opportunities. _Experimental & Molecular Medicine_, _52_(9), pp.1452-1465.

Giulietti, M., Occhipinti, G., Righetti, A., Bracci, M., Conti, A., Ruzzo, A., Cerigioni, E., Cacciamani, T., Principato, G. and Piva, F., 2018. Emerging biomarkers in bladder cancer identified by network analysis of transcriptomic data. _Frontiers in oncology_, _8_, p.450.

Horgan, R.P. and Kenny, L.C., 2011. 'Omic' technologies: genomics, transcriptomics, proteomics and metabolomics. _Obstetrician & Gynaecologist_, _13_(3).

Kosaka, N., Iguchi, H. and Ochiya, T., 2010. Circulating microRNA in body fluid: a new potential biomarker for cancer diagnosis and prognosis. _Cancer science_, _101_(10), pp.2087-2092.

Lowe, R., Shirley, N., Bleackley, M., Dolan, S. and Shafee, T., 2017. Transcriptomics technologies. _PLoS computational biology_, _13_(5), p.e1005457.

Nakagawa, M., Nakatani, F., Matsunaga, H., Seki, T., Endo, M., Ogawara, Y., Machida, Y., Katsumoto, T., Yamagata, K., Hattori, A. and Fujita, S., 2019. Selective inhibition of mutant IDH1 by DS-1001b ameliorates aberrant histone modifications and impairs tumor activity in chondrosarcoma. _Oncogene_, _38_(42), pp.6835-6849.

Puram, S.V., Tirosh, I., Parikh, A.S., Patel, A.P., Yizhak, K., Gillespie, S., Rodman, C., Luo, C.L., Mroz, E.A., Emerick, K.S. and Deschler, D.G., 2017. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. _Cell_, _171_(7), pp.1611-1624.

Siemelink, M.A. and Zeller, T., 2014. Biomarkers of coronary artery disease: the promise of the transcriptome. _Current cardiology reports_, _16_, pp.1-10.

Srivastava, R., 2022. Role of Transcriptomics in Precision Oncology. _Reports of Radiotherapy and Oncology_, _9_(1).

Van Rensburg, I.C. and Loxton, A.G., 2015. Transcriptomics: the key to biomarker discovery during tuberculosis?. _Biomarkers in medicine_, _9_(5), pp.483-495.

Yang, Z.H., Zheng, R., Gao, Y., Zhang, Q. and Zhang, H., 2014. Abnormal gene expression and gene fusion in lung adenocarcinoma with high-throughput RNA sequencing. _Cancer gene therapy_, _21_(2), pp.74-82.

Zhang, L., Xiao, H., Karlan, S., Zhou, H., Gross, J., Elashoff, D., Akin, D., Yan, X., Chia, D., Karlan, B. and Wong, D.T., 2010. Discovery and preclinical validation of salivary transcriptomic and proteomic biomarkers for the non-invasive detection of breast cancer. _PloS one_, _5_(12), p.e15573.****
